app/views/hyrax/leases/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Manage Leases</h1>
```

Comment:
Should these be adapted to look like our current styles, or is that something you want to take on in a separate PR?